matching-notification-api/src/main/java/org/phenotips/matchingnotification/internal/DefaultMatchingNotificationManager.java

```diff
@@ -89,7 +89,7 @@
 
             PatientMatchNotificationResponse notificationResult = this.notifier.notify(email);
 
-            if (notificationResult.isSuccessul()) {
+            if (notificationResult != null && notificationResult.isSuccessul()) {
```

Comment:
It looks like we did not test what happens when `notificationResult == null`. So this change is good. But what happens when we call `responses.add(notificationResult);` when notificationResult it is `null`, do we handle it correctly there?